src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -126,6 +126,12 @@
   <data name="DefaultSpecValueWarning" xml:space="preserve">
     <value>The value "{0}" for {1} is a sample value and should be removed. Replace it with an appropriate value or remove it and rebuild your package.</value>
   </data>
+  <data name="DependenciesGroupsForEachTFMHasCompatMatch" xml:space="preserve">
+    <value>Some TFMs declared in the dependencies group of the nuspec and the lib/ref folder have compatible matches, but not exact matches in the other location. Unless intentional, consult the list of actions below:{0}</value>
```

Comment:
do we use TFMs in other messages?
Should we use targetFramework?

Reply:
we use TFMs in the other rule I wrote, I believe